kale/loaddata/few_shot.py

```diff
@@ -0,0 +1,107 @@
+import os
+from typing import Any
+
+import numpy as np
+import torch
+from PIL import Image
+from torch.utils.data import Dataset
+
+
+class NWayKShotDataset(Dataset):
+    """
+    It is used to load data for N-way K-shot learning using Prototypical Networks.
```

Comment:
The doc can be more professional. Learn from others. `N` and `K` should be explained at the top maybe.